docs/modules/reference/pages/service-assurance/monitors/HttpMonitor.adoc

```diff
@@ -169,22 +176,28 @@ To test this behavior, configure the HttpMonitor as follows:
 [source, xml]
 ----
 <service name="HTTP-Allow-opennms.org" interval="300000" user-defined="false" status="on">
-  <parameter key="retry" value="1"/>
-  <parameter key="timeout" value="3000"/>
-  <parameter key="port" value="3128"/>
-  <parameter key="url" value="http://www.opennms.org/"/>
-  <parameter key="response" value="200-399"/>
+  <parameter key="retry" value="1"/> <1>
+  <parameter key="timeout" value="3000"/> <2>
+  <parameter key="port" value="3128"/> <3>
+  <parameter key="url" value="http://www.opennms.org/"/> <4>
+  <parameter key="response" value="200-399"/> <5>
 </service>
 
 <service name="HTTP-Block-tiktok.com" interval="300000" user-defined="false" status="on">
-  <parameter key="retry" value="1"/>
-  <parameter key="timeout" value="3000"/>
-  <parameter key="port" value="3128"/>
-  <parameter key="url" value="http://www.tiktok.com/"/>
-  <parameter key="response" value="400-599"/>
+  <parameter key="retry" value="1"/> <1>
+  <parameter key="timeout" value="3000"/> <2>
+  <parameter key="port" value="3128"/> <3>
+  <parameter key="url" value="http://www.tiktok.com/"/> <4>
+  <parameter key="response" value="400-599"/> <5>
 </service>
 
-<monitor service="HTTP-Allow-opennms.org" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor"/>
-<monitor service="HTTP-Block-tiktok.com" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor"/>
+<monitor service="HTTP-Allow-opennms.org" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor"/> <6>
+<monitor service="HTTP-Block-tiktok.com" class-name="org.opennms.netmgt.poller.monitors.HttpMonitor"/> <6>
 ----
+<1> Number of attempts to test a service’s status
+<2> Timeout for the isReachable method, in milliseconds.
+<3> A list of TCP ports to try to connect to.
+<4> URL to retrieve via the HTTP GET command.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.